examples/gcp_cli.py

```diff
@@ -107,7 +107,15 @@ def StartAnalysisVm(args: 'argparse.Namespace') -> None:
   attach_disks = []
   if args.attach_disks:
     attach_disks = args.attach_disks.split(',')
-
+    # Check if attach_disks parameter exists and if there
+    # are any empty entries.
+    if attach_disks and all(elements for elements in attach_disks):
+      pass
+    else:
+      print('error: parameter --attach_disks: {0:s}'.format(args.attach_disks))
+      return
```

Comment:
To make the code slightly simpler, you can use a guard statement:

```python
if not (attach_disks and all(elements for elements in attach_disks)):
  print(...)
  return

# Guard was passed proceed as normal
print('Starting analysis VM')
```

This is especially valuable here since you're just using `if / pass / else`, but also applies to `aws_cli.py`